src/EnergyPlus/EnergyPlusPgm.cc

```diff
@@ -395,27 +395,31 @@ EnergyPlusPgm( std::string const & filepath )
 	ReportOrphanFluids();
 	ReportOrphanSchedules();
 
-    if (runReadVars) {
+	if (runReadVars) {
 		std::string readVarsPath = exeDirectory + "ReadVarsESO" + exeExtension;
 		bool FileExists;
 		{ IOFlags flags; gio::inquire( readVarsPath, flags ); FileExists = flags.exists(); }
 		if (!FileExists) {
-			DisplayString("ERROR: Could not find ReadVarsESO executable: " + getAbsolutePath(readVarsPath) + "." );
-			exit(EXIT_FAILURE);
+			readVarsPath = exeDirectory + "PostProcess" + pathChar + "ReadVarsESO" + exeExtension;
+			{ IOFlags flags; gio::inquire( readVarsPath, flags ); FileExists = flags.exists(); }
```

Comment:
@nealkruis I know you just did the same as above, but why use gio::inquire to see if a file exists versus [FileSystem::pathExists](https://github.com/NREL/EnergyPlus/blob/6ce1086aa810e9d47f9fd46fa7c3905b2968f45f/src/EnergyPlus/FileSystem.hh#L36)? And I guess this is a more general question since I did use [FileSystem::pathExists in my unit testing fixture](https://github.com/NREL/EnergyPlus/blob/6ce1086aa810e9d47f9fd46fa7c3905b2968f45f/tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc#L339) instead of gio::inquire. @Myoldmopar @kbenne